Documentation/admin.rst

```diff
@@ -375,9 +375,7 @@ NodeSelector_ to the ``cilium-ds.yaml`` file like this:
 
 	spec:
 	  template:
-	    spec:
-	      nodeSelector:
-	        with-network-plugin: cilium
+            spec: nodeSelector: with-network-plugin: cilium
```

Comment:
probably doesn't matter but it looks like it's indented by 3 spaces instead of 2.